main.go

```diff
@@ -26,6 +26,7 @@ import (
 	// to ensure that exec-entrypoint and run can make use of them.
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
+	kfdefv1 "github.com/opendatahub-io/opendatahub-operator/apis/kfdef.apps.kubeflow.org/v1"
```

Comment:
To include kfdef in both this line and AddToScheme are only for thoese who are on  1.7.0(and pre) to upgrade to the first version of new operator, which means, if no old user on old operator, we can eventually cleanup these 2 lines?

Reply:
Those are just references to go modules. We can keep the logic or remove it eventually. It will handle usecases where there is no KfDef installed